2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html

```diff
@@ -0,0 +1,197 @@
+<!doctype html>
+
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+
+        <title>Tools for OMERO Users</title>
+
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+        <link rel="stylesheet" href="reveal-js/css/reveal.min.css">
+        <link rel="stylesheet" href="reveal-js/css/theme/default.css" id="theme">
+
+        <!-- OME presentation overrides of the default theme -->
+        <link rel="stylesheet" href="css/ome.css">
+
+        <!-- For syntax highlighting -->
+        <link rel="stylesheet" href="reveal-js/lib/css/zenburn.css">
+
+        <!-- If the query includes 'print-pdf', use the PDF print sheet -->
+        <script>
+            document.write( '<link rel="stylesheet" href="reveal-js/css/print/' + ( window.location.search.match( /print-pdf/gi ) ? 'pdf' : 'paper' ) + '.css" type="text/css" media="print">' );
+        </script>
+
+    </head>
+
+    <body class="ome_theme">
+
+        <div class="reveal">
+
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- title page -->
+                <section>
+                    <h1>Tools for <br>OMERO Users</h1>
+                    <p>
+                        OME Users meeting<br>
+                        30th May 2018<br>
+                        Will Moore
+                    </p>
+                </section>
+                
+                <section>
+                    <h2>Outline</h2>
+                    <ul>
+                        <li>"Tools": OMERO apps and CLI clients</li>
+                        <li>Why increased focus on tools?</li>
+                        <li>OMERO.server release process</li>
+                        <li>Advantages of decoupled clients</li>
+                        <li>Testing of decoupled clients</li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>"Tools": OMERO apps and CLI clients</h2>
+
+                    <ul>
+                        <li>OMERO.scripts</li>
+                        <li>Web Apps:
+                            <ul>
+                                <li>figure</li>
+                                <li>iviewer</li>
+                                <li>FPBioImage</li>
+                                <li>webtagging</li>
```

Comment:
need to differentiate between the ones we maintain and others.
webtagging is not under our controls. (not release for 1 year)
there is also omero-forms